solutions/pandas_tests/pytest.ini

```diff
@@ -0,0 +1,11 @@
+# pytest.ini
```

Comment:
Add comments on how you get the list of markers, making easier for future contributors to maintain.

Reply:
Added comment to pytest.ini